packages/ember-simple-auth-cookie-store/lib/simple-auth-cookie-store/stores/cookie.js

```diff
@@ -96,6 +96,18 @@ export default Base.extend({
   _syncDataTimeout: null,
 
   /**
+    @property _renewExpirationTimeout
+    @private
+  */
+  _renewExpirationTimeout: null,
```

Comment:
Yeah, the library never uses "_" prefixes for private members so that should be consistent.